middleware/index.ts

```diff
@@ -168,6 +168,22 @@ function createContext(id: string, logger: Logger, reqContext: any, accessToken?
     return context;
 }
 
+/**
+ * TODO: replace this with salesforce-sdk api call to get secret
+ * @param name
+ * @param key
+ * @param logger
+ */
+function getSecret(name, key, logger: Logger) : string {
+    try {
+        let data = fs.readFileSync(`/platform/services/${name}/secret/${key}`);
+        return data.toString();
```

Comment:
Could `data` be null/undefined?

Reply:
I think it could be and empty byte array, should never be null/undefined (readFileSync would throw an error in that case).

Reply:
I'm hoping to replace this with salesforce sdk, so I don't have to worry about this.
But yes... I should add some null checking...
JSYK, from CLI, when you create a secret key, they do not allow you to have secretkey=, you must have something on the RHS of the =